snippets/javascript/class-object.md

```diff
@@ -0,0 +1,39 @@
+# Text encryptor
+
+You are making a text encryptor. It should take multiple words and output a
+combined version, where each word is separated by a dollar sign `$`.
+
+For example, for the words "hello", "how", "are", "you", the output should be
+`"$hello$how$are$you$"`.
+
+```js
+class Add {
+  constructor(...words) {
+    this.words = words;
+  }
+  print() {
+    let finalString = "";
+    for (let index = 0; index < this.words.length; index++) {
+      finalString += "$" + this.words[index];
```

Comment:
here also `this`.

Reply:
The JavaScript `this` keyword refers to the object it belongs to. Here, `this` refers to the `Add`.